src/bindings/js/node/package.json

```diff
@@ -28,17 +28,17 @@
     "install_runtime": "npm run download_runtime -- --ignore-if-exists"
   },
   "devDependencies": {
-    "@eslint/js": "^9.30.1",
-    "@types/node": "^22.13.4",
-    "eslint": "^9.30.1",
-    "eslint-config-prettier": "^10.1.5",
-    "eslint-plugin-prettier": "^5.5.1",
-    "globals": "^16.3.0",
-    "mocha": "^11.7.1",
-    "prettier": "3.6.2",
+    "@eslint/js": "^9.39.1",
+    "@types/node": "^22.19.1",
+    "eslint": "^9.39.1",
+    "eslint-config-prettier": "^10.1.8",
+    "eslint-plugin-prettier": "^5.5.4",
+    "globals": "^16.5.0",
+    "mocha": "^11.7.5",
+    "prettier": "^3.7.3",
```

Comment:
We agreed to use specific version of prettier to avoid formatting changes due to new releases of prettier. What changed?